.env.test

```
@@ -90,3 +90,27 @@ SIGNIN_LOCKOUT_SECONDS=900
+OAUTH_GOOGLE_REDIRECT_URI=http://localhost/api/auth/social/google/callback
+
+OAUTH_FACEBOOK_CLIENT_ID=test_facebook_client_id
+OAUTH_FACEBOOK_CLIENT_SECRET=test_facebook_client_secret
+OAUTH_FACEBOOK_REDIRECT_URI=http://localhost/api/auth/social/facebook/callback
+
+OAUTH_TWITTER_CLIENT_ID=test_twitter_client_id
+OAUTH_TWITTER_CLIENT_SECRET=test_twitter_client_secret
+OAUTH_TWITTER_REDIRECT_URI=http://localhost/api/auth/social/twitter/callback
+
+OAUTH_STATE_TTL_SECONDS=600
+OAUTH_PROVIDER_HTTP_CONNECT_TIMEOUT_MS=1500
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OAUTH_PROVIDER_HTTP_CONNECT_TIMEOUT_MS key should go before the OAUTH_STATE_TTL_SECONDS key

(UnorderedKey)